xml/System.Text.Json/JsonProperty.xml

```diff
@@ -113,10 +139,24 @@
         <Parameter Name="text" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="text">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="text">The text to compare against.</param>
+        <summary>Compares <paramref name="text" /> to the name of this property.</summary>
+        <returns><see langword="true" /> if the name of this property matches <paramref name="text" />,
```

Comment:
```suggestion
        <returns><see langword="true" /> if the name of this property matches <paramref name="text" />; otherwise <see langword="false" />.</returns>
```